examples/common/python/utility/signature.py

```diff
@@ -224,7 +228,7 @@ def generate_client_signature(self, input_json_str,
         input_json_params["sessionKeyIv"] = ''.join(format(i, '02x') for i in session_iv)
 
         encrypted_session_key_str = ''.join(format(i, '02x') for i in encrypted_session_key)
-        self.__encrypt_workorder_indata(input_json_params, session_key,
+        self.encrypt_workorder_indata(input_json_params, session_key,
```

Comment:
Why did we make it global function ? could find this function getting called elsewhere ?

Reply:
Since these functions are already there in signature.py, i am making use of it by changing to global function. signature.py is utility module for client to do signature and hashing. Any client can make use of it.